README.md

```diff
@@ -20,3 +20,51 @@ Rather than matching graphical tiles with the representation of logical tiles on
     * These alpha masks are used to combine each corner's terrain texture into a single tile.
 * In future this will be much more customisable, using Perlin noise to generate noisy gradients and boundaries betwen corners.
 * Currently all tile generation is done through a command-line application configured using a JSON file. In the future, this should be migrated to a GUI interface which allows the user to alter tile generation parameters and immediately see the range of possible results.
+
+# How can it build?
```

Comment:
Oh, it should be probably more like "How can I build the Wangscape?", didn't notice.